README.md

```diff
@@ -36,52 +41,46 @@ development stage (no stable release available).
 
 To compile LibrePCB, you need the following software components:
 - g++ >= 4.8, MinGW >= 4.8, or Clang >= 3.3 (C++11 support is required)
-- Qt >= 5.2 (http://www.qt.io/download-open-source/)
-- `libglu1-mesa-dev` (`sudo apt-get install libglu1-mesa-dev`)
-- `zlib` / `zlib-dev` (http://www.zlib.net/)
-- `OpenSSL` (https://www.openssl.org/)
+- [Qt](http://www.qt.io/download-open-source/) >= 5.2
+- [zlib](http://www.zlib.net/)
+- [OpenSSL](https://www.openssl.org/)
 
-#### Installation on Ubuntu 14.04 and later
+#### Installation on Debian/Ubuntu/Mint
 
 ```bash
-sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator libglu1-mesa-dev zlib1g zlib1g-dev openssl
+sudo apt-get install g++ qt5-default qttools5-dev-tools qt5-doc qtcreator \
+libglu1-mesa-dev zlib1g zlib1g-dev openssl
 ```
 
-#### Installation on ArchLinux
-
-You can install [librepcb-git](https://aur.archlinux.org/packages/librepcb-git/) from the AUR.
-
-#### Installation on Mac OS X 10.11
-
-1. Install Xcode through the app store and start it at least once 
-   (for the license)
-2. Install [homebrew](https://github.com/Homebrew/brew) (**the** package 
-   manager)
-3. Install *qt5*
+#### Installation on Mac OS X
+1. Install Xcode through the app store and start it at least once (for the license)
+2. Install [homebrew](https://github.com/Homebrew/brew) (**the** package manager)
+3. Install *qt5*: `brew update && brew install qt5`
+4. Make the toolchain available: `brew unlink qt && brew link --force qt5`
 
-        brew update && brew install qt5
+#### Installation on Windows
 
-4. Make the toolchain available (you can undo it by linking against *qt*
-   again)
+Download and run the 
+[Qt for Windows (MinGW) installer](http://download.qt.io/official_releases/qt/5.8/5.8.0/qt-opensource-windows-x86-mingw530-5.8.0.exe) 
+from [here](https://www.qt.io/download-open-source/). LibrePCB does not compile 
+with MSVC, so you must install following components with the Qt installer:
+- The MinGW compiler itself
+- The Qt libraries for MinGW
 
-        brew unlink qt
-        brew link --force qt5
+### Cloning
 
-5. [Compile everything](#user-content-using-qmake-and-make)
-6. Run the program
+It's important to clone the repository recursively to get all submodules too:
 
-        open ./build/generated/mac/librepcb.app
-
-#### Installation in a docker container
-
-To build and run LibrePCB in a [docker](https://www.docker.com/) container (which is pretty cool!), check out [these instructions](https://github.com/LibrePCB/LibrePCB/tree/master/dev/docker).
+```bash
+git clone --recursive https://github.com/LibrePCB/LibrePCB.git && cd LibrePCB
+```
```

Comment:
I'd like to keep the readme short, and because this information is redundant (same can be done with removing and re-cloning the repository) it feels like spam... ;)